src/authorisation/routeMethods.js

```diff
@@ -182,43 +182,45 @@ let app;
 // success ist die Funktion, die aufgerufen wird, wenn die Authorisierung geglückt ist.
 // TODO: Fehlerfälle
 function isAuthorised(req, res, success) {
-    app.oauth.authorise()(req, res, (authResult) => {
+    const token = req.get("Authorization").slice("Bearer ".length)
```

Comment:
@janzse hier fehlen not null Abfragen.  Wenn es keinen token header gibt, stürzt das ab.